pkg/util/exec/exec.go

```diff
@@ -313,3 +313,12 @@ func assertErrorType(err error) string {
 
 	return ""
 }
+
+// ExtractExitCode attempts to get the exit code from the error returned by an Executor function.
+// This should also work for any errors returned by the golang os/exec package.
+func ExtractExitCode(err error) (int, error) {
+	if ee, ok := err.(*exec.ExitError); ok {
+		return ee.ExitCode(), nil
+	}
+	return 0, errors.Errorf("error %#v is not an ExitError", err)
```

Comment:
Do you really need an error? The caller just use `exitCode == 22` so just returning 0 when not `ExitError` seems sufficient.

Reply:
I think that's true, but the error makes it more clear and forces users to handle the error. I think it's the better pattern to follow, IMO. `0` is a valid return code for applications, even if that shouldn't happen when there is an ExitError.